incubator/hnc/pkg/controllers/object_controller.go

```diff
@@ -284,14 +292,38 @@ func (r *ObjectReconciler) onDelete(ctx context.Context, log logr.Logger, nnm ty
 	return nil
 }
 
+func (r *ObjectReconciler) ensureCanPropagate(nsNm string, inst *unstructured.Unstructured) bool {
+	r.Forest.Lock()
+	defer r.Forest.Unlock()
+	ns := r.Forest.Get(nsNm)
+	if !ns.Exists() {
+		return false
+	}
+	//TODO TAYLOR
+	var conditionString string
+	if len(strings.Split(r.GVK.String(), ".")) > 1 {
```

Comment:
I don't understand this line? There will _always_ be a GVK for _every_ object. However, for legacy reason, the "core" objects (like `Pod` or `Secret`) have an empty string as the group. Eg, a pod "bar" in namespace "foo" should have the key `/v1/pod/foo/bar`, while a replica set "rs" in the same namespace would have the key `apps/v1/pod/foo/rs`.